tests/shap/test_log.py

```diff
@@ -0,0 +1,93 @@
+import mlflow
+import shap
+import numpy as np
+import sklearn
+from mlflow.utils.environment import _mlflow_conda_env
+
+
+def get_sklearn_conda_env():
+    """
+    :return: The default Conda environment for MLflow Models produced by calls to
+             :func:`save_model()` and :func:`log_model()`.
+    """
+    pip_deps = ["shap=={}".format(shap.__version__)]
+
+    return _mlflow_conda_env(
+        additional_conda_deps=["scikit-learn={}".format(sklearn.__version__)],
+        additional_pip_deps=pip_deps,
+        additional_conda_channels=None,
+    )
+
+def test_sklearn_log_explainer():
+    """
+    Tests mlflow.shap log_explainer with mlflow serialization of the underlying model
+    """
+
+    with mlflow.start_run() as run:
+
+        run_id = run.info.run_id
+
+        X, y = shap.datasets.boston()
+        model = sklearn.ensemble.RandomForestRegressor(n_estimators=100)
+        model.fit(X, y)
+    
+        explainer_original = shap.Explainer(model.predict, X, algorithm='permutation')
+        shap_values_original = explainer_original(X[:5])
+
+        mlflow.shap.log_explainer(explainer_original, "test_explainer")
+
+        explainer_new = mlflow.shap.load_explainer("runs:/" + run_id + "/test_explainer")
```

Comment:
```suggestion
        explainer_loaded = mlflow.shap.load_explainer("runs:/" + run_id + "/test_explainer")
```

Can we rename the variable name here? I think `new` is a bit misleading.

Reply:
Renamed variable